<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DingTest_dlg</class>
 <widget class="QDialog" name="DingTest_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/new/prefix2/Ico/ren.ico</normaloff>:/new/prefix2/Ico/ren.ico</iconset>
  </property>
  <widget class="QPushButton" name="Btn_Measure">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>220</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>测量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Quit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>370</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Del">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>440</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_EditTestItems">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>120</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改参数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_NewTestItems">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>70</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新建项目</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_IsZero">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>170</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>校零</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="group_TestCH">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>选择通道</string>
   </property>
   <widget class="QCheckBox" name="checkBox_CH1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>通道一</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_CH2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>通道二</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_CH3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>通道三</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>120</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>选择检测项目</string>
   </property>
   <widget class="QComboBox" name="comboBox_TestItems">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Btn_Printer">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>440</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>631</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Display">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>显示</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox_TestItems</tabstop>
  <tabstop>checkBox_CH1</tabstop>
  <tabstop>checkBox_CH2</tabstop>
  <tabstop>checkBox_CH3</tabstop>
  <tabstop>Btn_NewTestItems</tabstop>
  <tabstop>Btn_EditTestItems</tabstop>
  <tabstop>Btn_IsZero</tabstop>
  <tabstop>Btn_Measure</tabstop>
  <tabstop>Btn_Printer</tabstop>
  <tabstop>Btn_Del</tabstop>
  <tabstop>Btn_Quit</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
